File: Iniciante/1961/1961.py
# -*- coding: utf-8 -*-

p = int(input().split(" ")[0])

values = input().strip().split(" ")

win = True
prev = int(values[0])

for value in values[1:]:
    value = int(value)
    min = prev - p
    max = prev + p
    if value < min or value > max:
        win = False
        break
    prev = value

if win:
    print("YOU WIN")
else:
    print("GAME OVER")

File: Iniciante/1961/1961.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);

    int p = Integer.parseInt(scan.nextLine().split(" ")[0]);

    String[] values = scan.nextLine().trim().split(" ");

    boolean win = true;
    int prev = Integer.parseInt(values[0]);

    for (int i = 1; i < values.length; i++) {
      int value = Integer.parseInt(values[i]);

      int min = prev - p;
      int max = prev + p;
      if (value < min || value > max) {
        win = false;
        break;
      }

      prev = value;
    }

    if (win) {
      System.out.println("YOU WIN");
    } else {
      System.out.println("GAME OVER");
    }
  }
}
